base_ubl/models/ubl.py

```diff
@@ -784,10 +784,16 @@ def ubl_parse_incoterm(self, delivery_term_node, ns):
         return {}
 
     def ubl_parse_product(self, line_node, ns):
+        # GTIN schemeID is 0160
         barcode_xpath = line_node.xpath(
-            "cac:Item/cac:StandardItemIdentification/cbc:ID[@schemeID='GTIN']",
+            "cac:Item/cac:StandardItemIdentification/cbc:ID[@schemeID='0160']",
```

Comment:
I think you can simplify it by using `"cac:Item/cac:StandardItemIdentification/cbc:ID[@schemeID='0160'|@schemeID='GTIN']",`

Can you test? (It would be awesome to use an old file and a new file in tests in order to ensure that everything works fine